monitoring: Add a link for hot-reloading (#1271)
fluent/fluent-bit-docs: administration/monitoring.md

| /api/v1/health             | Fluent Bit health check result                                                                                                                                     | String                |

> | /api/v2/metrics            | Internal metrics per loaded plugin                                                                                                                                 | cmetrics' text format |

Reviewer: ```suggestion
| /api/v2/metrics            | Internal metrics per loaded plugin                                                                                                                                 | [cmetrics text format](https://github.com/fluent/cmetrics) |
```

| /api/v2/metrics/prometheus | Internal metrics per loaded plugin ready to be consumed by a Prometheus Server                                                                                     | Prometheus Text 0.0.4 |

> | /api/v2/reload             | Execute hot reloading or get the status of hot reloading. In more details, please refer to the [hot-reloading doc](hot-reload.md).                                 | JSON                  |

Reviewer: ```suggestion
| /api/v2/reload             | Execute hot reloading or get the status of hot reloading. For more details, please refer to the [hot-reloading documentation](hot-reload.md).                                 | JSON                  |
```
